tests/snappi_tests/multidut/pfcwd/files/pfcwd_multidut_basic_helper.py

```diff
@@ -100,10 +100,10 @@ def run_pfcwd_basic_test(api,
 
     if trigger_pfcwd:
         """ Large enough to trigger PFC watchdog """
-        pfc_storm_dur_sec = ceil(detect_time_sec + poll_interval_sec + 0.1)
+        pfc_storm_dur_sec = 4 * ceil(detect_time_sec + poll_interval_sec + 0.1)
```

Comment:
could you pls explain why this change is needed? I believe it's passing in T0/T1 with this settings.

Reply:
@sdszhang , we discussed internally. Based on the discussion, I will try this script without changes, and see how close it gets to failure. I will update the PR based on the findings.